File: program/program/convert-feet-to-miles/convertFeetToMiles.scala
import scala.io.StdIn._;

object feetToMiles {
  def main(args:Array[String]):Unit={
    println("Enter the feet value : ");
    var feet = readLine();
    var miles = feet.toDouble / 5280;
    println(feet+" feet is "+miles+" miles");
  }
}

File: program/program/convert-feet-to-miles/convert_feet_to_miles.pl
print "Enter the feet  : ";
my $feet = <STDIN> ;

my $res = feetToMiles();
print "Equivalent Miles : $res";

sub feetToMiles
{
    my ($x) = @_;
    my $res = $feet / 5280;
    return $res ;   
}
